documentation/book/cluster-operator.adoc

```diff
@@ -722,6 +765,26 @@ The resources created by the Cluster Operator into the {ProductPlatformName} clu
 [[kafka_connect_configuration_json_config]]
 ===== Kafka Connect configuration
 
+The `logging` field allows configuration of loggers. These loggers are:
+`
+log4j.rootLogger
+connect.root.logger.level
+log4j.logger.org.apache.zookeeper
+log4j.logger.org.I0Itec.zkclient
+log4j.logger.org.reflections
+`
```

Comment:
Similar to the comment above, the newlines separating the logging names appear to be removed when this text is rendered